deep_bottleneck/plotter/informationplane_movie.py

```diff
@@ -27,31 +27,57 @@ def generate(self, measures_summary):
     def plot(self, measures_summary):
 
         measures = measures_summary['measures_all_runs']
+        activations_summary = measures_summary['activations_summary']
 
         os.makedirs('plots/', exist_ok=True)
 
         plt.set_cmap("hsv")
-        fig, ax = plt.subplots()
+        fig, (ax_infoplane, ax_accuracy) = plt.subplots(2,1, figsize=(6, 9),
+                                                        gridspec_kw={'height_ratios': [2, 1]})
+
         if self.dataset == 'datasets.mnist' or self.dataset == 'datasets.fashion_mnist':
-            ax.set(xlim=[0, 14], ylim=[0, 3.5])
+            ax_infoplane.set(xlim=[0, 14], ylim=[0, 3.5])
         else:
-            ax.set(xlim=[0, 12], ylim=[0, 1])
+            ax_infoplane.set(xlim=[0, 12], ylim=[0, 1])
 
-        ax.set(xlabel='I(X;M)', ylabel='I(Y;M)')
+            ax_infoplane.set(xlabel='I(X;M)', ylabel='I(Y;M)')
 
-        scatter = ax.scatter([], [], s=20, edgecolor='none')
-        text = ax.text(0, 1.05, "", fontsize=12)
+        scatter = ax_infoplane.scatter([], [], s=20, edgecolor='none')
+        text = ax_infoplane.text(0, 1.05, "", fontsize=12)
 
         num_layers = measures.index.get_level_values(1).nunique()
         layers_colors = np.linspace(0, 1, num_layers)
 
-        writer = FFMpegWriter(fps=10)
+        acc = []
+        val_acc = []
+        acc_line,  = ax_accuracy.plot([], [], 'b', label="training accuracy")
```

Comment:
This syntax for single tuple unpacking is a bit hard to read. Better use `[a] = t`, see https://stackoverflow.com/questions/33161448/getting-only-element-from-a-single-element-list-in-python